### PR TITLE
Made the links in the toaster message clickable

### LINKED ISSUE #367: Need to provide a option to click on errors link in the toast message.
For example: "You cannot add/modify the time entries. Please visit https://www.google.com for more information.". The "https://www.google.com" will be a href \ html say having a grid of error details.

Screenshot 2021-07-26 at 6 28 45 PM

### PR DESCRIPTION
**Pull request by amaldinesh7**
Fixes #367 

- Added package linkify to make the links in the toaster message clickable.

@edwinbbu _a Please review.
cc: @goutham-subramanyam

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +83,8 @@
     "raw-loader": "^4.0.2",
     "react-color": "^2.19.3",
     "react-hotkeys-hook": "^3.3.2",
+    "react-linkify": "^1.0.0-alpha",
+    "react-popper": "^2.2.5",
```
Comment (edwinbbu) [inline package.json:87]: @amaldinesh7 _a `react-popper` package is not being used. Can you remove that.

**Comment by edwinbbu** [comment]
@amaldinesh7 Can you add example for v1 toastr in v1 playground as well to verify the changes.

File: package.json (hunk 1 above)
Comment (amaldinesh7) [reply package.json]: @edwinbbu  _a Removed

**Comment by amaldinesh7** [comment]
> @amaldinesh7 Can you add example for v1 toastr in v1 playground as well to verify the changes.

Its already there @edwinbbu

**Comment by unnitallman** [comment]
@edwinbbu @amaldinesh7 A high priority task in Aceinvoice is waiting for this issue to be completed. It would be great if we could get this done at the earliest. Thank you.

**Pull request merged**